bazel-mode.el

```diff
@@ -21,10 +21,23 @@
 ;;
 ;;; Code:
 
+(defconst bazel-mode-syntax-table
+  (let ((table (make-syntax-table)))
+	;; single and multiple line comment start
+	(modify-syntax-entry ?# "<" table)
+	;; single line comment end
+	(modify-syntax-entry ?\n ">" table)
+	table)
+  "Syntax table for `bazel-mode'.")
+
 ;;;###autoload
 (define-derived-mode bazel-mode prog-mode "Bazel"
-  "Major mode for editing Bazel BUILD and WORKSPACE files.")
-
+	"Major mode for editing Bazel BUILD and WORKSPACE files."
+	(setq comment-start "# ")
```

Comment:
IIUC these variables aren't buffer-local by default (check with <kbd>C-h v comment-start</kbd>), so you probably should use `setq-local` here.

Reply:
Fixed. You're correct. I saw setq-local used in https://www.emacswiki.org/emacs/SampleMode and setq https://www.emacswiki.org/emacs/DerivedMode. But the check you suggested shows that it's not buffer-local by default.